.github/workflows/bump-image-tags.yaml

```diff
@@ -0,0 +1,43 @@
+name: Bump Image Tags
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1" # Run at 00:00 UTC every Monday
+
+env:
+  team_reviewers: 2i2c-org:tech-team
+
+jobs:
+  bump-image-tags:
+    runs-on: ubuntu-latest
+    environment: github-app-token-generator
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # For each new config_path to monitor, add a new item in this matrix.
+          # The Action can read multiple paths to images, but not multiple config files.
+          - name: "pangeo-hubs common singleuser image"
+            config_path: "config/clusters/pangeo-hubs/common.values.yaml"
+            values_paths: ".basehub.jupyterhub.singleuser.image"
+
+    steps:
+      # We want tests to be run on the Pull Request that gets opened by the next step,
+      # so we generate a token from a GitHub App that would allow this.
```

Comment:
```suggestion
      # so we generate a token from a GitHub App that would allow this.
      By default, secrets.GITHUB_TOKEN is prevented from triggering
      other workflows.
```